### ISSUE TITLE
Update or replace django-extra-views

### ISSUE THREAD

**Issue by Jawayria** [edx/upgrades#4]
We use the package in [django-extra-views](https://github.com/AndrewIngram/django-extra-views) in edx/ecommerce. But it hasn't yet added support for Django 3.2. Please follow the guidance in https://openedx.atlassian.net/wiki/spaces/AC/pages/3036972032/Handling+Outdated+Dependencies to resolve the problem this poses for the Open edX Django 3.2 upgrade.

**Comment by MaferMazu** [comment]
This library supports Django3.2, you can see more details of the tests here https://github.com/AndrewIngram/django-extra-views/blob/master/tox.ini

It would be necessary to test the use of that dependency in edx/ecommerce. I could work on it

**Comment by jmbowman** [comment]
The latest release on PyPI didn't claim support for Django 3.2, but it looks like the very next commit added testing for it without needing any code changes.  So it looks like you're correct in that we just need to upgrade the dependency in order to resolve this.

**Comment by MaferMazu** [comment]
Ok. I have a question, @jmbowman . What would be the next step to take? Do a PR to the `django-32-upgrade` branch by changing the requirements from `django-extra-views == 0.11.0` to `django-extra-views == 0.14.0`?

**Comment by natabene** [comment]
@MaferMazu Thank you so much for looking into this. @jmbowman Could you please help us out?

**Comment by jmbowman** [comment]
@MaferMazu Yes, I think that's basically all that's needed, although it looks like there are some breaking changes in those versions that may need to be accounted for: https://github.com/AndrewIngram/django-extra-views/blob/master/CHANGELOG.rst .  The frustrating thing here is that upstream already dealt with this, and we're kicking off a project to get back onto an upstream django-wiki release, but it probably won't be done by the time we need the Django 3.2 upgrade finished.
